index.php

```diff
@@ -37,13 +63,22 @@
                             <option value="4">Type</option>
                         </select>
                     </div>
+                    </form>
                 </div>
             </div>
 
 
             <?php
 
-            $sql = "SELECT jobs.*, user.company_name, user.company_site FROM jobs JOIN user ON user.id_user = jobs.id_user where status = 'active'";
+            // $sql = "SELECT * FROM `jobs` WHERE ( CONVERT(`title` USING utf8) LIKE '%продавач%' AND `status` = 'active') LIMIT 0,2";
+            if (!empty($_GET['search'])) {
+                $search = $_GET['search'];
+                $sql = "SELECT jobs.*, user.company_name, user.company_site FROM `jobs` JOIN user ON user.id_user = jobs.id_user WHERE ( CONVERT(`title` USING utf8) LIKE '%" . $search . "%' AND `status` = 'active') LIMIT 0,20";
```

Comment:
this is hard to read because of the long query.

I would move this to a function and get the results, do one request and based on the function arguments, we can add the active check or skip it

Reply:
I've moved it to a function